app/src/main/java/net/gsantner/markor/activity/DocumentShareIntoFragment.java

```diff
@@ -116,22 +122,58 @@ public void onViewCreated(final @NonNull View view, final Bundle savedInstanceSt
         }
     }
 
-    @Override
-    public String getFragmentTag() {
-        return FRAGMENT_TAG;
+    private CheckBox addCheckBoxToToolbar() {
```

Comment:
Instead of having the share link be an item in the list, move it to the toolbar. This is cleaner and takes even less space.

![image](https://github.com/gsantner/markor/assets/1940940/10933953-8cbd-4f72-93c8-8f61722b9dfd)